<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_clientClass</class>
 <widget class="QWidget" name="tcp_clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcp_client</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>主机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_hostLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_portLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>接受的信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnConnect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Send">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcp_client.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_BtnConnect</sender>
   <signal>clicked()</signal>
   <receiver>tcp_clientClass</receiver>
   <slot>on_ConnectButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_Send</sender>
   <signal>clicked()</signal>
   <receiver>tcp_clientClass</receiver>
   <slot>sendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_ConnectButton_clicked()</slot>
  <slot>sendMessage()</slot>
 </slots>
</ui>
